### PR TITLE
Refs #356 — Attempt to fix eslint new configuration errors.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: test/functional_test.js [hunk 1]
```diff
@@ -1392,10 +1392,7 @@ function runOnPrefix(apiPrefix) {
             conf.set('hekaMetrics', hekaMetrics);
             sandbox.stub(hekaLogger, "info", function(op, log) {
               log.op = op;
-              try {
-                _logs.push(log);
-              } catch (e) {
```
Comment (ametaireau) [inline test/functional_test.js]: I wonder why we had an exception here at first...

Comment (ametaireau) [inline test/functional_test.js]: This was done by @tarekziade in https://github.com/mozilla-services/loop-server/commit/eef15f3217e18d58348c80e63ffb738a0d9acbc9 I don't quite understand why this is from the commit message. I would be tempted to just remove it.

@tarekziade any insights?

Comment (Natim) [inline test/functional_test.js]: > I would be tempted to just remove it.

Yes it is just what I did :)

Comment (ametaireau) [inline test/functional_test.js]: Okay, scratch it for now. We can go back later if needed.

**Pull request merged**